### PR TITLE
add links for edugator on main page

### PR DESCRIPTION
**Pull request by marcgabe15**
## What does this PR do?
This PR adds in Links for the vertical navigation bar on Edugator using react router dom.

### CODE REVIEW

**Review by AqSlz — APPROVED** [review]
LGTM

File: src/pages/LandingPage/LandingHome.tsx [hunk 1]
```diff
@@ -95,14 +96,20 @@ function LandingHome() {
...
             </Typography>
             <ButtonHolder>
-              <Button
-                variant="contained"
-                color="primary"
-                size="large"
-                style={{ marginRight: 15 }}
+              <Link
+                to={{
+                  pathname: "/code",
```
Comment (joshuadargan) [inline src/pages/LandingPage/LandingHome.tsx:101]: why not Routes.Code?

**Review by joshuadargan — COMMENTED** [review]
One comment otherwise lgtm

**Review by joshuadargan — APPROVED** [review]
lgtm

**Review by AqSlz — APPROVED** [review]

**Pull request merged**